inspirehep/base/searchext/mappings/hep.json

```diff
@@ -57,15 +57,32 @@
                         }
                     }
                 },
+                "abstracts": {
+                    "type": "object",
+                    "properties": {
+                        "value": {
+                            "type": "string",
+                            "copy_to": ["abstract", "global_fulltext"]
+                        }
+                    }
+                },
+                "abstract": {
+                    "type": "string"
+                },
                 "accelerator_experiments": {
                     "type": "object",
                     "properties": {
                         "experiment": {
                             "type": "string",
-                            "index": "not_analyzed"
+                            "index": "not_analyzed",
```

Comment:
Shouldn't we split the experiment as well? E.g. would be great to match CERN-LHC-LHCb even when the user has typed `LHCb`.

I think this could be the occasion to have both a `.raw` subfield not analyzed and an analyzed one.